thunder/benchmarks/conftest.py

```diff
@@ -0,0 +1,52 @@
+import platform
+import psutil
+from typing import Any
+
+from asvdb import utils, BenchmarkInfo, BenchmarkResult, ASVDb
+
+
+def sanitize_params(benchmark_params: list[tuple[str, Any]]) -> list[tuple[str, Any]]:
+    sane_params = []
+    for k, v in benchmark_params:
+        if k == "executor":
+            sane_params += [(k, str(v).split()[1])]
+        else:
+            sane_params += [(k, v)]
+    return sane_params
+
+
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
How would a developer use this functionality? Can we provide some examples of disabling it, enabling it, and using its options (if any) in a comment?